index.html

```diff
@@ -0,0 +1,301 @@
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas dolores porro totam facere ipsam assumenda aspernatur animi corrupti! Nisi vel nesciunt saepe corrupti impedit dicta ipsam nam, dolores deleniti quibusdam.</p>
+        <button class="btn">Lorem, ipsum dolor.</button>
+      </div>
+      <div class="guide tier-3">
+        <h1>Lorem, ipsum dolor.</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas dolores porro totam facere ipsam assumenda aspernatur animi corrupti! Nisi vel nesciunt saepe corrupti impedit dicta ipsam nam, dolores deleniti quibusdam.</p>
+        <button class="btn">Lorem, ipsum dolor.</button>
+      </div>
+      <div class="guide tier-3">
+        <h1>Lorem, ipsum dolor.</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas dolores porro totam facere ipsam assumenda aspernatur animi corrupti! Nisi vel nesciunt saepe corrupti impedit dicta ipsam nam, dolores deleniti quibusdam.</p>
+        <button class="btn">Lorem, ipsum dolor.</button>
+      </div>
+      <div class="guide tier-2 newsletter">
+        <div class="fake-pic pic-circle"></div>
+        <h1>Lorem, ipsum.</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero expedita cum veniam. Rerum consequatur minus deleniti, eos explicabo ut laudantium aliquid!</p>
+        <button class="btn">Lorem, ipsum dolor.</button>
+        <div class="small-box">Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit eos sed quibusdam?</div>
+      </div>
+    </article>
+    <section class="more-link">
+      <span>Lorem ipsum dolor sit.</span>
+    </section>
+    <footer class="tier-3">
+      <h4>Lorem ipsum dolor sit amet.</h4>
+      <ul class="list">
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+        <li>Lorem, ipsum.</li>
+      </ul>
+      <div class="footer-content">
+      </div>
+    </footer>
```

Comment:
- [ ] The footer contains centered links, so please use proper HTML tags to create footer links and apply proper CSS attributes to center them.

![image](https://user-images.githubusercontent.com/60184961/91844126-912e4b80-ec5f-11ea-827b-a7252a85f8b4.png)